scripts/resultsSum.pl

```diff
@@ -260,47 +260,47 @@ sub resultReporter {
 		my $hudsonUrl = $ENV{'HUDSON_URL'};
 		if ((!defined $hudsonUrl) || ($hudsonUrl eq '')) {
 			$hudsonUrl = "https://ci.adoptopenjdk.net/";
-		}
-		my $vendorParam =  "";
-		if ($jdkVendor ne '') {
-			$vendorParam = "&JDK_VENDOR=" . $jdkVendor;
-		}
-		# TODO: Revisit this and make it more generic and allow all potentially
-		#       applicable Grinder job parameters to be made available
-		my $jenkinsParam = "";
-		if ( defined $ENV{'JCK_GIT_REPO'} ) {
-			$jenkinsParam .= "&JCK_GIT_REPO="        . $ENV{'JCK_GIT_REPO'};
-		}
-		if ( defined $ENV{'SDK_RESOURCE'} ) {
-			$jenkinsParam .= "&SDK_RESOURCE="        . $ENV{'SDK_RESOURCE'};
-		}
-		if ( defined $ENV{'CUSTOMIZED_SDK_URL'} ) {
-			$jenkinsParam .= "&CUSTOMIZED_SDK_URL="  . $ENV{'CUSTOMIZED_SDK_URL'};
-		}
-		if ( defined $ENV{'CUSTOMIZED_SDK_URL_CREDENTIAL_ID'} ) {
-			$jenkinsParam .= "&CUSTOMIZED_SDK_URL_CREDENTIAL_ID=" . $ENV{'CUSTOMIZED_SDK_URL_CREDENTIAL_ID'};
-		}
-		if ( defined $ENV{'UPSTREAM_JOB_NAME'} ) {
-			$jenkinsParam .= "&UPSTREAM_JOB_NAME="   . $ENV{'UPSTREAM_JOB_NAME'};
-		}
-		if ( defined $ENV{'UPSTREAM_JOB_NUMBER'} ) {
-			$jenkinsParam .= "&UPSTREAM_JOB_NUMBER=" . $ENV{'UPSTREAM_JOB_NUMBER'};
-		}
+			my $vendorParam =  "";
+			if ($jdkVendor ne '') {
+				$vendorParam = "&JDK_VENDOR=" . $jdkVendor;
+			}
+			# TODO: Revisit this and make it more generic and allow all potentially
+			#       applicable Grinder job parameters to be made available
+			my $jenkinsParam = "";
+			if ( defined $ENV{'JCK_GIT_REPO'} ) {
+				$jenkinsParam .= "&JCK_GIT_REPO="        . $ENV{'JCK_GIT_REPO'};
+			}
+			if ( defined $ENV{'SDK_RESOURCE'} ) {
+				$jenkinsParam .= "&SDK_RESOURCE="        . $ENV{'SDK_RESOURCE'};
+			}
+			if ( defined $ENV{'CUSTOMIZED_SDK_URL'} ) {
+				$jenkinsParam .= "&CUSTOMIZED_SDK_URL="  . $ENV{'CUSTOMIZED_SDK_URL'};
+			}
+			if ( defined $ENV{'CUSTOMIZED_SDK_URL_CREDENTIAL_ID'} ) {
+				$jenkinsParam .= "&CUSTOMIZED_SDK_URL_CREDENTIAL_ID=" . $ENV{'CUSTOMIZED_SDK_URL_CREDENTIAL_ID'};
+			}
+			if ( defined $ENV{'UPSTREAM_JOB_NAME'} ) {
+				$jenkinsParam .= "&UPSTREAM_JOB_NAME="   . $ENV{'UPSTREAM_JOB_NAME'};
+			}
+			if ( defined $ENV{'UPSTREAM_JOB_NUMBER'} ) {
+				$jenkinsParam .= "&UPSTREAM_JOB_NUMBER=" . $ENV{'UPSTREAM_JOB_NUMBER'};
+			}
```

Comment:
We can remove Line 267 - 287.

Reply:
Hi @llxia, I have removed line 267 - 287, please review when you are free.